src/main/java/org/alliancegenome/curation_api/services/associations/alleleAssociations/AlleleGeneAssociationService.java

```diff
@@ -47,10 +49,14 @@ public class AlleleGeneAssociationService extends BaseAssociationDTOCrudService<
 	@Inject
 	AlleleDAO alleleDAO;
 	@Inject
+	AlleleService alleleService;
+	@Inject
 	NoteDAO noteDAO;
 	@Inject
 	GeneDAO geneDAO;
 	@Inject
+	GeneService geneService;
```

Comment:
It doesn't look like the gene or allele services are getting used?

Reply:
Good spot - will remove